lib/system/io.nim

```diff
@@ -507,10 +507,12 @@ const
     # should not be translated.
 
 when defined(posix) and not defined(nimscript):
+  type
+    Mode {.importc: "mode_t", header: "<sys/types.h>".} = cushort
+      # BUGFIX: was cint
```

Comment:
You can't just say `BUGFIX` without even explaining it. This belongs to the commit message also.

Reply:
fixed by deferring to https://github.com/nim-lang/Nim/pull/13941